test/server-livereload.test.js

```diff
@@ -326,4 +326,32 @@ describe('Helix Server with Livereload', () => {
       await project.stop();
     }
   });
+
+  it('livereload of a query definition should not cause an unhandled rejection when no last URL exists', async () => {
+    const rejects = [];
+    process.on('unhandledRejection', (reason) => {
+      rejects.push(reason);
+    });
```

Comment:
interestingly, `mocha` does not display this handled rejection and will not cause the test to fail, either.